task.py

```diff
@@ -0,0 +1,33 @@
+# Найдите сумму цифр трехзначного числа.
+
+# n = input("Введите трехзначное число: ")
+# n = int(n)
+ 
+# d1 = n % 10
+# n = n // 10
+# d2 = n % 10
+# n = n // 10
+ 
+# print("Сумма цифр числа:", n + d1 + d2)
+
+
+#Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# s = int(input())
+# print((s//6), ((s//6)*4), (s//6))
+
+
+#Вы пользуетесь общественным транспортом? Вероятно, вы расплачивались за проезд и получали билет с номером. Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех. Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. Вам требуется написать программу, которая проверяет счастливость билета.
+#n = input().strip()
+#print(['Обычный', 'Счастливый билет'][sum(list(map(int, n[:3])))==sum(list(map(int, n[3:])))])
```

Comment:
Тернарный оператор, круто. Переводить в список не нужно функция `sum` и так разберется с итератором, который возвращает функция `map`.
```suggestion
#print(['Обычный', 'Счастливый билет'][sum(map(int, n[:3])) == sum(map(int, n[3:]))])
```